.github/scripts/oss-build.functions.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -euo pipefail
+
+function get_hz_dist_zip() {
+  local hz_variant=$1
+  local hz_version=$2
+  local suffix=$3
+
+  # The slim is an artifact with a classifier, need to add `-` there
+  if [[ -n "${hz_variant}" ]]; then suffix="-${hz_variant}"; fi
+
+  if [[ "${hz_version}" == *"SNAPSHOT"* ]]
+  then
+      # DI-95 - Do not rely on the OSS distribution zip in the hazelcast-docker PR builder
+      # https://hazelcast.atlassian.net/browse/DI-95
+      aws s3 presign s3://hazelcast/distribution-snapshot/hazelcast-"${hz_version}""${suffix}".zip --expires-in 600
```

Comment:
I'm pretty sure those quotes are wrong here
```suggestion
      aws s3 presign s3://hazelcast/distribution-snapshot/hazelcast-${hz_version}${suffix}.zip --expires-in 600
```

Reply:
it's quoted [because of this](https://www.shellcheck.net/wiki/SC2086).

I'm testing it now.

Reply:
actually it seems it works. but more idiomatic way would be to do it like this:
```shell
aws s3 presign "s3://hazelcast/distribution-snapshot/hazelcast-${hz_version}${suffix}.zip" --expires-in 600
```

Reply:
93f73b6bd4630c9c8e280d39e93eb4786a3c9cc1